src/App.tsx

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string | EventTarget | null,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
+  };
+
+  componentDidMount(): void {
+    document.documentElement.addEventListener('keydown', this.keyHandler);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keydown', this.keyHandler);
+  }
+
+  keyHandler = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: `The last pressed key is [${event.key}]` });
```

Comment:
it would be better to store the `event.key` only and add check in render method
```suggestion
    this.setState({ pressedKey: event.key });
```